tf-modules/archive/api.tf

```diff
@@ -0,0 +1,117 @@
+locals {
+  api_port_substring = var.api_port == null ? "" : ":${var.api_port}"
+  api_uri            = var.api_url == null ? "https://${aws_api_gateway_rest_api.api.id}.execute-api.${data.aws_region.current.name}.amazonaws.com${local.api_port_substring}/${var.api_gateway_stage}/" : var.api_url
+  api_redirect_uri   = "${local.api_uri}token"
+}
+
+resource "aws_cloudwatch_log_group" "api" {
+  name              = "/aws/lambda/${aws_lambda_function.api.function_name}"
+  retention_in_days = 30
+}
+
+resource "aws_lambda_function" "api" {
+  depends_on = [aws_iam_role.lambda_api_gateway]
+
+  function_name    = "${var.prefix}-ApiEndpoints"
+  filename         = "${path.module}/../../packages/api/dist/app/lambda.zip"
+  source_code_hash = filebase64sha256("${path.module}/../../packages/api/dist/app/lambda.zip")
+  handler          = "index.handler"
+  role             = aws_iam_role.lambda_api_gateway.arn
+  runtime          = "nodejs8.10"
+  timeout          = 20
+  environment {
+    variables = {
+      AccessTokensTable            = var.dynamo_tables.AccessTokens
+      AsyncOperationTaskDefinition = aws_ecs_task_definition.async_operation.arn
+      AsyncOperationsTable         = var.dynamo_tables.AsyncOperations
+      BulkDeleteLambda             = aws_lambda_function.bulk_delete.arn
+      CMR_ENVIRONMENT              = var.cmr_environment
+      CollectionsTable             = var.dynamo_tables.Collections
+      EARTHDATA_BASE_URL           = "${replace(var.urs_url, "//$/", "")}/" # Make sure there's a trailing slash
+      EARTHDATA_CLIENT_ID          = var.urs_client_id
+      EARTHDATA_CLIENT_PASSWORD    = var.urs_client_password
+      ES_HOST                      = var.elasticsearch_hostname
+      EcsCluster                   = var.ecs_cluster_name
+      EmsDistributionReport        = aws_lambda_function.ems_distribution_report.arn
+      EmsIngestReport              = aws_lambda_function.ems_ingest_report.arn
+      EmsProductMetadataReport     = aws_lambda_function.ems_product_metadata_report.arn
+      ExecutionsTable              = var.dynamo_tables.Executions
+      GranulesTable                = var.dynamo_tables.Granules
+      IndexFromDatabaseLambda      = aws_lambda_function.index_from_database.arn
+      KinesisInboundEventLogger    = data.aws_lambda_function.kinesis_inbound_event_logger.arn
+      OAUTH_PROVIDER               = var.oauth_provider
```

Comment:
I think this is also missing the oauth_user_group (from 0565146), as well as the Launchpad stuff from  https://github.com/nasa/cumulus/pull/1136.   Same as the ref above re: timeout.